snapcraft.yaml

```diff
@@ -342,6 +350,23 @@ parts:
       echo "ac_add_options --with-mozilla-api-keyfile=$SNAPCRAFT_STAGE/mozilla-desktop-geoloc-api.key" >> $MOZCONFIG
     override-prime: ''
 
+  debug-symbols:
+    plugin: nil
+    build-packages:
+      - python3
+      - python3-virtualenv
+    after:
+      - firefox
+    override-pull: |
+      if [ -f "$SNAPCRAFT_PROJECT_DIR/symbols-upload-token" ]; then
+        export SYMBOLS_ARCHIVE=$(find ../../firefox/build/obj-*/dist/ -type f -name "firefox-*.crashreporter-symbols.zip")
```

Comment:
It would be cleaner to have the firefox part copy the symbols file(s) to some shared place (e.g. `$SNAPCRAFT_STAGE`), rather than hardcode `../../firefox/build/`, which is a snapcraft implementation detail.

Reply:
Yes, I did not knew about that, will do.